pages/home.jsx

```diff
@@ -109,7 +113,7 @@ var BlogSection = React.createClass({
         return;
       }
       this.setState({
-        featuredPost: data.featuredPosts,
+        featuredPost: data.featuredPost,
```

Comment:
um sorry for being picky, i just thought this should be singular instead of plural :speak_no_evil: